internal/template/templates/views/feed_entries.html

```diff
@@ -88,7 +88,7 @@ <h3>{{ t "alert.feed_error" }}</h3>
                     {{ if ne .Feed.Icon.IconID 0 }}
                         <img src="{{ route "icon" "iconID" .Feed.Icon.IconID }}" width="16" height="16" loading="lazy" alt="{{ .Feed.Title }}">
                     {{ end }}
-                    <a href="{{ route "feedEntry" "feedID" .Feed.ID "entryID" .ID }}">{{ .Title }}</a>
+                    <a href="{{ route "feedEntry" "feedID" .Feed.ID "entryID" .ID }}{{ if not $.showOnlyUnreadEntries }}?all{{ end }}">{{ .Title }}</a>
```

Comment:
I don't think there is an agreement on
whether
`/category/<category_id>/entry/<entry_id>?unread=1`
or
`/category/<category_id>/unread/entry/<entry_id>`

`/category/<category_id>/entry/<entry_id>?unread=1`

Looks easier here

Reply:
Or 'unread/category/id/entries' ?
As we already have 'unread' route and we can keep 'category/id/entries' part consistent.